vms/platformvm/blocks/executor/acceptor.go

```diff
@@ -176,10 +104,20 @@ func (a *acceptor) ApricotAtomicBlock(b *blocks.ApricotAtomicBlock) error {
 			err,
 		)
 	}
+
+	a.ctx.Log.Trace(
```

Comment:
Note: I also changed these to `Trace` so that we could better observe these values (without all the p2p messages)